compiler/noirc_printable_type/src/lib.rs

```diff
@@ -282,11 +307,17 @@ impl<F: AcirField> std::fmt::Display for PrintableValueDisplay<F> {
                 let mut display_iter = values.iter();
                 let re = Regex::new(r"\{([a-zA-Z0-9_]+)\}").map_err(|_| std::fmt::Error)?;
```

Comment:
Maybe this logic could be done with regex, just going char by char and checking if we get `{` but not `{` after it... but for now I left it as is, maybe it's fine.